frontend/packages/kubevirt-plugin/integration-tests/tests/vm.action.clone.scenario.ts

```diff
@@ -281,50 +277,48 @@ describe('Test clone VM.', () => {
     it(
       'ID(CNV-1744) Test clone VM with URL source and Cloud Init.',
       async () => {
-        const ciVMConfig = {
-          name: `ci-${testName}`,
-          namespace: testName,
-          description: `Default description ${testName}`,
-          provisionSource: cloudInitVmProvisionConfig,
-          storageResources: [rootDisk],
-          networkResources: [],
-          flavorConfig: basicVMConfig.flavorConfig,
-          operatingSystem: basicVMConfig.operatingSystem,
-          workloadProfile: basicVMConfig.workloadProfile,
-          startOnCreation: false,
-          cloudInit: cloudInitCustomScriptConfig,
-        };
+        const vmWithCloudInit = new VMBuilder(basicVM)
+          .setProvisionSource(provisionSources.URL)
+          .setDisks([rootDisk])
+          .setFlavor(flavorConfigs.Tiny)
+          .setCloudInit(cloudInitCustomScriptConfig)
+          .generateName()
+          .build();
+
         const expectedDisks = [
           rootDisk,
-          { name: 'cloudinitdisk', size: '', interface: 'VirtIO', storageClass: '-' },
+          {
+            name: 'cloudinitdisk',
+            size: '',
+            interface: 'VirtIO',
+            storageClass: '-',
+            drive: DISK_DRIVE.Disk,
+          },
         ];
 
-        const wizard = new Wizard();
-        const ciVM = await wizard.createVirtualMachine(ciVMConfig);
-        await withResource(leakedResources, ciVM.asResource(), async () => {
-          await ciVM.action(VM_ACTION.Clone);
+        await vmWithCloudInit.create();
+        await withResource(leakedResources, vmWithCloudInit.asResource(), async () => {
+          await vmWithCloudInit.clone();
           await cloneDialog.clone();
           const clonedVM = new VirtualMachine({
-            name: `${ciVMConfig.name}-clone`,
-            namespace: ciVM.namespace,
+            name: `${vmWithCloudInit.name}-clone`,
```

Comment:
I think it would be interesting.
Would you suggest passing the clone method a VMBuilder object?
Or should we create something like CloneVMBuilder? The thing is that only name, namespace, description and startOnCreation can be changed, so using VMBuilder could lead to creating a VirtualMachine with data that don't reflect reality

Reply:
I just meant a VirtualMachine to have clone method - without builders. Does it makes sense?

Reply:
Yes, I'll add it